src/sidebar/components/Annotation/AnnotationHeader.js

```diff
@@ -115,10 +115,10 @@ function AnnotationHeader({
 
   return (
     <header>
-      <div className="hyp-u-layout-row--align-baseline hyp-u-horizontal-spacing--2 u-wrap-reverse">
+      <div className="hyp-u-layout-row--align-baseline hyp-u-horizontal-spacing--2 flex-wrap-reverse">
         {annotationIsPrivate && !isEditing && (
           <Icon
-            classes="u-icon--xsmall"
+            classes="text-tiny"
```

Comment:
Our `Icon` components are sized at `1em` currently so setting text size is a valid way to size them. This is inelegant and disorganized, however; I intend to revisit icon sizing (it's on the proverbial list).

Reply:
Re-using text sizes for icons also makes clear the intent about the size of icons compared to text on the same line, which is nice.